CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+* The `role_revoke` function now takes a `%Role{}`, and a `%Group{}` instead of
+  names. It returns a `%Group{}` with the associated roles as an attribute. The
+  original `role_revoke` can still be used for users.
```

Comment:
This is actually still querying by name, you just need to provide a struct that has that key of name. Given that the way you call it is still the same, do you think we should still leave it?

Seems like we should be forging the way to the "final" api is my reason for not leaving in the old stuff.